api/client/stats_helpers.go

```diff
@@ -24,6 +24,7 @@ type containerStats struct {
 	NetworkTx        float64
 	BlockRead        float64
 	BlockWrite       float64
+	PidsCurrent      uint64
```

Comment:
Maybe something got lost in a rebase because I have used it and it works

On Tuesday, March 8, 2016, Tõnis Tiigi <notifications@github.com> wrote:

> In api/client/stats_helpers.go
> <https://github.com/docker/docker/pull/18697#discussion_r55464136>:
>
> > @@ -24,6 +24,7 @@ type containerStats struct {
> >  	NetworkTx        float64
> >  	BlockRead        float64
> >  	BlockWrite       float64
> > +	PidsCurrent      uint64
>
> I only meant that setting of this value before pritning seems to be
> missing on the client side. My commit is based on the containerd branch so
> the daemon side is different there.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/18697/files#r55464136>.
>


-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>